lib/app/modules/alarmRing/controllers/alarm_ring_controller.dart

```diff
@@ -112,7 +112,14 @@ class AlarmControlController extends GetxController {
   @override
   void onInit() async {
     super.onInit();
-
+    if (currentlyRingingAlarm.value.deleteAfterGoesOff == true) {
+      IsarDb.deleteAlarm(currentlyRingingAlarm.value.isarId);
+      
+      FirestoreDb.deleteOneTimeAlarm(
```

Comment:
You can check if the alarm is a shared alarm or not, unless it's shared it wouldn't be stored on Firestore.